### DISCUSSION TITLE
Getting an error 404 Not found and message "Not Found"

### DISCUSSION THREAD

**Discussion by Ankit8420** [octokit/octokit.js#1827] [Q&A] [answered]
When i try to create a new repo by using the Postman above message is shown.
Url: https://api.github.com/user//repos
Even i pass the body also to the request, below is the body of the request:

```json
{
  "name": "Create new repo",
  "description": "Create new repo by API",
  "homepage": "https://github.com",
  "private": false
}
```

Also pass the username and password in the Authorization. Also try to change the password with the token.
Please advice what is wrong

**Comment by gr2m** [comment, marked as answer]
you have a double `//` in the URL, might that be the problem?

**Reply by Ankit8420** [reply]
Thanks it worked
